sunpy/visualization/animator/image.py

```diff
@@ -160,10 +160,10 @@ class ImageAnimatorWCS(ImageAnimator):
     """
     def __init__(self, data, wcs=None, image_axes=[-1, -2], unit_x_axis=None, unit_y_axis=None,
                  axis_ranges=None, **kwargs):
-        if not isinstance(wcs, (astropy.wcs.WCS, BaseLowLevelWCS)):
-            raise ValueError("wcs data should be provided.")
+        if not isinstance(wcs, BaseLowLevelWCS):
```

Comment:
Why this change?

Reply:
`astropy.wcs.WCS` inherits `BaseLowLevelWCS` so it was redundant.